src/metabase/email/messages.clj

```diff
@@ -333,14 +333,11 @@
       (not= :table (render/detect-pulse-chart-type card nil result-data))
       (no "we've determined it should not be rendered as a table")
 
-      (= (count (take render.body/cols-limit cols)) render.body/cols-limit)
```

Comment:
If I understand correctly `cols-limit` always is `##INF` so think it's okay to remove this as a temporarily fix , but wdyt we remove this `cols-limit` variable all together?
Cam has a TODO about this https://github.com/metabase/metabase/blob/91bd47814a4f07a5aa2df589b91492d56f92fa7b/src/metabase/pulse/render/body.clj#L58 and I think it's safe to remove it now.